### ISSUE TITLE
Wont work with multiple screens

### ISSUE THREAD

**Issue by XayOn** [akshat46/perpetual-zen#11]
Function handle_tag_icons receives an empty selected_tag for some reason.
I don't have enough awesome background to debug it, but it might be because of the screen[1] call on banners.lua:22

I added an if that allows awesome to start for now just by disabling spotify widget (wich also won't start) but it still produces some artifacts.

**Comment by XayOn** [comment]
[artifact](https://imgur.com/a/nH43Pn6)

**Comment by akshat46** [comment]
Yes that line is the issue. I have added a quick fix to a [separate branch](https://github.com/akshat46/perpetual-zen/tree/multiple-monitor-fix). 
I don't have access to a multi-screened setup so I can't test the behavior of it. It should add a distinct set of workspaces and their tags to any additional screen. I probably won't try to achieve anything more for this as of now since I don't have any experience with multiple screens to figure out what would be an intuitive behavior for tags in such circumstances.

**Comment by XayOn** [comment]
I don't have two screens atm, I'll try the branch and confirm it tomorrow at work.
I think the "use whatever monitor the current focus is at" is intuitive enough.
